src/app/routes/applications/new-role/new-role.component.ts

```diff
@@ -618,6 +617,10 @@ export class NewRoleComponent implements OnInit, AfterViewInit {
     this.goNextStep();
   }
 
+  isLastStep() {
+    return this.stepper?.steps?.length - 1 === this.stepper?.selectedIndex;
```

Comment:
Would we want `isLastStep` to return `true` if `this.stepper` is undefined? I think this is what we would get here as `undefined === undefined` is `true`, I believe. So maybe worth checking if `this.stepper` is undefined?
```suggestion
    return this.stepper && this.stepper?.steps?.length - 1 === this.stepper?.selectedIndex;
```

Reply:
It might be when initializing component/view. That's why i added elvis operator. It should work without it as well.